ArgoProject/src/systems/HealthSystem.h

```diff
@@ -21,6 +21,8 @@ namespace app::sys
 	public: // Public Static Functions
 	public: // Public Member Functions
 		virtual void update(app::time::seconds const & dt) override;
+		void checkPlayerHealth();
+		void checkDestructibleHealth();
```

Comment:
the two functions `checkPlayerHealth()` and `checkDestructibleHealth()` should be private member functions.